### ISSUE TITLE
Does it support mysql?

### ISSUE THREAD

**Issue by FAKERINHEART** [gamegos/cesi#19]

**Comment by hxpro** [comment]
No. Cesi use database only for store userinfo like username, password and type. There is no relevant reason for mysql support.

**Comment by kozdincer** [comment]
Yes @hxpro Also we think remove sqlite/authentication support and give it to external reverse proxy.

**Comment by hxpro** [comment]
Absolutely agree, this is useless functionality.
